src/FSharp.AWS.DynamoDB/TableContext.fs

```diff
@@ -2096,3 +2149,12 @@ module Scripting =
         member t.UpdateProvisionedThroughput(provisionedThroughput: ProvisionedThroughput) : unit =
             let spec = Throughput.Provisioned provisionedThroughput
             t.UpdateTableIfRequiredAsync(spec) |> Async.Ignore |> Async.RunSynchronously
+
+/// Helpers for working with <c>TransactWriteItemsRequest</c>
+module TransactWriteItemsRequest =
```

Comment:
I think this should be either renamed or namespaced within `Transaction` to align with the top level API that can cause it to happen

i.e. make this 
```
module Transaction =
```